adhocteam/Head-Start-TTADP
# Download of activity reports

## Description of change

 * Activity reports can now be downloaded in a bulk, instead of 10 at a time
 * Replaced Buttons on the filter to look like actual buttons
 * Tag shows the number of selected reports
 * Reports menu button allows downloading of all reports or just selected reports
 * Add ability to download all `MyAlerts`

## How to test

1) Pull down
2) Approve some reports (over 10)
3) Filter and download approved reports. Verify you are able to download over 10 reports and can download the reports you set a filter for
4) Match the UI against the [design](https://preview.uxpin.com/ad995fd722bd26c0f9a1c8a115f1b41833637308#/pages/138463475)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-103

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
